### PR TITLE
add filter to README

### PR DESCRIPTION
**Pull request by ldanilek**

### CODE REVIEW

File: packages/convex-helpers/README.md [hunk 1]
```diff
@@ -66,7 +68,11 @@ const posts = await asyncMap(
     const comments = await getManyFrom(db, "comments", "postId", post._id);
     // many-to-many via join table
     const categories = await getManyViaOrThrow(
-      db, "postCategories", "categoryId", "postId", post._id
+      db,
+      "postCategories",
+      "categoryId",
+      "postId",
+      post._id
```
Comment (ianmacartney) [inline packages/convex-helpers/README.md:75]: I'm guessing you have some VSCode setting that formats markdown with relevant formatters? 

I slightly prefer the one-liner aesthetic for a readme, but this is fine

File: packages/convex-helpers/README.md [hunk 2]
```diff
@@ -309,3 +327,37 @@ const balanceAndEmail = pick(Account.withoutSystemFields, ["balance", "email"]);
...
+
+See the [guide on Stack](https://stack.convex.dev/complex-filters-in-convex)
+for an analysis of complex filters on Convex.
+
+The `filter` helper composes with `ctx.db.query` to apply arbitrary TypeScript
+or JavaScript filters to a database query.
+
+Examples:
+
+```javascript
```
Comment (ianmacartney) [inline packages/convex-helpers/README.md:341]: can you include the import statement so it's easy to copy? It's often not clear where to import things from and auto-complete for imports hasn't been working well for me for helpers for some reason

**Review by ianmacartney — APPROVED** [review]

File: packages/convex-helpers/README.md [hunk 3]
```diff
@@ -309,3 +327,38 @@ const balanceAndEmail = pick(Account.withoutSystemFields, ["balance", "email"]);
...
+See the [guide on Stack](https://stack.convex.dev/complex-filters-in-convex)
+for an analysis of complex filters on Convex.
+
+The `filter` helper composes with `ctx.db.query` to apply arbitrary TypeScript
+or JavaScript filters to a database query.
+
+Examples:
+
+```js
+import { filter } from "convex-helpers/server/filter";
```
Comment (ianmacartney) [inline packages/convex-helpers/README.md:342]:
```suggestion
import { filter } from "convex-helpers/server/filter";

```

File: packages/convex-helpers/README.md (hunk 1 above)
Comment (ldanilek) [reply packages/convex-helpers/README.md:75]: Yeah we can revert the formatting change, it was automatic

**Pull request merged**